device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C224O8/buffers_defaults_objects.j2

```diff
@@ -0,0 +1,310 @@
+{#
```

Comment:
is it possible to use a symbol link to buffers_defaults_objects.j2 2700? Like what you did for C256S2

Reply:
Done